docs-2.0/7.data-security/1.authentication/3.role-list.md

```diff
@@ -65,7 +65,7 @@ The privileges of roles and the nGQL statements that each role can use are liste
   |Read data|Y|Y|Y|Y|Y|`GO`, `SET`, `PIPE`, `MATCH`, `ASSIGNMENT`, `LOOKUP`, `YIELD`, `ORDER BY`, `FETCH VERTICES`, `Find`, `FETCH EDGES`, `FIND PATH`, `LIMIT`, `GROUP BY`, `RETURN`|
   |Write data|Y|Y|Y|Y||`INSERT VERTEX`, `UPDATE VERTEX`, `INSERT EDGE`, `UPDATE EDGE`, `DELETE VERTEX`, `DELETE EDGES`, `DELETE TAG`|
   |Show operations|Y|Y|Y|Y|Y|`SHOW`, `CHANGE PASSWORD`|
-
+  |Job|Y|Y|Y|Y||`SUBMIT JOB COMPACT`、`SUBMIT JOB FLUSH`、`SUBMIT JOB STATS`、`STOP JOB`、`RECOVER JOB`|
```

Comment:
Maybe you need to add a blank line below, there may have some formatting problems.

Reply:
Fixed， thanks